README.md

```diff
@@ -97,49 +89,23 @@ import { HttpClientModule } from '@angular/common/http';
 })
 export class AppModule {
   constructor(matIconRegistry: MatIconRegistry, domSanitizer: DomSanitizer){
-    matIconRegistry.addSvgIconSet(domSanitizer.bypassSecurityTrustResourceUrl('./assets/mdi.svg')); // Or whatever path you placed mdi.svg at
+    matIconRegistry.addSvgIconSet(
+      domSanitizer.bypassSecurityTrustResourceUrl('./assets/mdi.svg')
+    );
   }
 }
 ```
 
-Usage:
+2. The SVG icons can then be used with [`MatIcon`](https://material.angular.io/components/icon/api#MatIcon)'s
+`svgIcon` attribute as shown below:
 
 ```html Example Usage
-
-<!-- Icon by itself -->
-<mat-icon svgIcon="android" aria-label="Android icon"></mat-icon>
-<!-- Icon button -->
-<a mat-icon-button href="https://android.com" matTooltip="Go to Android.com" aria-label="Go to Android.com">
-  <mat-icon svgIcon="android" aria-label="Android icon"></mat-icon>
-</button>
-<!-- You can also combine an icon and text together -->
-<button mat-button>
-  <mat-icon svgIcon="code-tags" aria-label="Code icon"></mat-icon>
-  View source
-</button>
-
-```
-
-Please also add the following class to your styles (`styles.css`) to solve the problem where an icon isn't aligned properly when used in a menu item:
-
-```css styles.css
-
-button.mat-menu-item {

```

Comment:
Are you sure that this CSS is obsolete ?

Otherwise, it could be still interesting to keep somewhere, for instance it could in a "Troubleshooting" section.